galaxy/tools/verify/interactor.py

```diff
@@ -65,20 +73,20 @@ def __getitem__(self, item):
             return item
 
 
-def stage_data_in_history(galaxy_interactor, tool_id, all_test_data, history, force_path_paste=False):
+def stage_data_in_history(galaxy_interactor, tool_id, all_test_data, history):
```

Comment:
Wrong merge, should be reverted.

Reply:
Yeah - I noticed that above... I'm working on it. 😞